### ISSUE TITLE
consider adding <style> to the supported tags

### ISSUE THREAD

**Issue by EsoHasInsomnia** [googlefonts/picosvg#216]
example:
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><style>.st0{fill:none;}</style><path class="st0" d="M0 0h24v24H0z"/></svg>

throws an error

**Comment by EsoHasInsomnia** [comment]
I thought style is not supported but i found this
https://github.com/googlefonts/picosvg/blob/727fee175e3523326f15cb02ab4a4e504a8f6e88/src/picosvg/svg.py#L250

**Comment by anthrotype** [comment]
yes, style _attributes_ are supported,  but not `<style>` elements currently. I'm not sure how involved it would be to support those too. Probably we'd need a full CSS stylesheet parser.

**Comment by vXBaKeRXv** [comment]
"Unable to convert to picosvg: " + ",".join(nano_violations)
ValueError: Unable to convert to picosvg: BadElement: /svg[0]/style[0]
